### PR TITLE
[#27] Add linter, fix style issues found by the linter

### PR DESCRIPTION
**Pull request by robertoaloi**

### CODE REVIEW

File: src/aws_credentials.erl [hunk 1]
```diff
@@ -8,7 +8,9 @@
 -behaviour(gen_server).
 
 %% As per
-%% http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html#instance-metadata-security-credentials
+%% http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/
+%% iam-roles-for-amazon-ec2.html#instance-metadata-security-credentials
+%%
```
Comment (mrallen1) [inline src/aws_credentials.erl:13]: URLs should not get whitespace in the middle of them, I should be able to copy and paste them

**Review by mrallen1 — CHANGES_REQUESTED** [review]
I don't really know much about elvis tbh, so I don't even know if this is possible to implement...

**Comment by robertoaloi** [comment]
@mrallen1 Indeed, found the option that allows us to do that.

**Comment by robertoaloi** [comment]
Merging, since the only comment has been addressed.

**Pull request merged**